mentors/src/data_pipeline/utils.py

```diff
@@ -0,0 +1,5 @@
+VIDEO_FILE = "video.mp4"
```

Comment:
seems like right now this file contains only constants, so probably `constants.py` is a better name